### DISCUSSION TITLE
Symbolic Links

### DISCUSSION THREAD

**Discussion by ryanbrackney** [AbdBarho/stable-diffusion-webui-docker#267] [General]
[This is maybe a general docker questions, but I'll ask here in case.]

I'd like to make a symbolic link to a singular model folder so I don't need to have multiple GB files for every SD application on Win11. However, I can't seem to get those links to show up in the automatic111 UI within docker, even though I'm putting the link inside the data/StableDiffusion folder.  I've tried creating a SymbolicLink, HardLink, and Junction with Windows Powershell, and a symbolic link from within Ubuntu/WSL. None show up within the container. 

Anyone have any luck / experience getting this to work?

**Comment by AbdBarho** [comment]
I think regardless of soft or hardlinks, both the source and target must be reachable in the container.

As an example, if your symlink points to `../../Desktop/models/sd1.5.ckpt`, then the automatic ui will try to follow that path to the model, and if the folder structure in the container does not match that of the link / host, the symlink won't work.


One option to work around this issue is to symlink in the other direction, i.e. symlink from `data/models` to other apps, this should work as long as the other apps run on the host (not containerized), but it is up to you if you see this as a fitting solution to your problem.

#268 vaguely goes in the same direction, of having better globally shared data.

**Reply by ryanbrackney** [reply]
Thanks! I'm not sure why I didn't think of just specifying another volume directly before, it's pretty obvious in hindsight.
